src/test.ts

```diff
@@ -25,49 +25,49 @@ import * as ts from "typescript";
 
 import { Replacement } from "./language/rule/rule";
 import * as Linter from "./linter";
+import { createMultiMap } from "./rules/no-unused/utils";
```

Comment:
Needed to change the test runner to be able to test files that import each other -- these must end in `.ts` and not `.lint` or TypeScript can't resolve the modules.